src/main/java/core/dao/GetDataFromFileCsv.java

```diff
@@ -0,0 +1,41 @@
+package core.dao;
+
+import core.service.ActivitiesStrategy;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class GetDataFromFileCsv implements GetDataFromFile {
+    private static final int ACTIVITY_TYPE = 0;
+    private static final int FRUIT_TYPE = 1;
+    private static final int AMOUNT_OF_FRUITS = 2;
+    private final ActivitiesStrategy activitiesStrategy;
+
+    public GetDataFromFileCsv(ActivitiesStrategy activitiesStrategy) {
+        this.activitiesStrategy = activitiesStrategy;
+    }
+
+    @Override
+    public String getData(String fileName) {
```

Comment:
this method does 5 different things: reads lines from file, parses CSV lines, puts data to storage, gets it back from storage and creates report based on the data